interfaces/builtin/serial_port_test.go

```diff
@@ -173,6 +180,9 @@ slots:
         interface: serial-port
         path: /dev/ttyS
     bad-path-10:
+        interface: serial-port
+        path: /dev/ttySC
```

Comment:
"path: /dev/ttySCZ" would also be a good test to add. I can do that in a follow-up commit.